RELEASE.md

```diff
@@ -17,6 +17,7 @@
 * Added minimal, black-compatible flake8 configuration to the project template.
 * Moved `isort` and `pytest` configuration from `<project_root>/setup.cfg` to `<project_root>/pyproject.toml`.
 * Fixed a bug where extra parameters were incorrectly passed from `KedroSession` to `KedroContext`.
+* Fixed a typo in the [Kedro spaceflights tutorial](/docs/source/03_tutorial/01_spaceflights_tutorial.md)
```

Comment:
Generally speaking this is not required as we tend to add the notes only about the changes that may impact the usage of Kedro, while typo fixes in the documentation can be done silently 🙂 
```suggestion
* Fixed a typo in the [Kedro spaceflights tutorial](/docs/source/03_tutorial/01_spaceflights_tutorial.md).
```